components/Avatar/avatar.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Avatar</title>
+    <link rel="stylesheet" href="avatar.css">
+    <link rel="shortcut icon" href="images/favicon-32x32.png" type="image/x-icon">
+    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css">
+</head>
+<body>
+    <header class="header">
+        <nav class="navbar">
+            <!-- logo -->
+            <a  class="logo" href="/"><div class="nav-logo">
+                LILAC UI
+            </div>
+            </a>
+    
+            <!-- nav links -->
+            <div class="nav-links">
+                <a href="docs.html" class="nav-items"><span>Docs</span></a>
+                <a href="https://github.com/muskan-gupta-0803/LilacUI" class="nav-items"><span>Github</span></a>
+                <a href="#" class="nav-items"><span>
+                    <i class="fas fa-sun"></i>
+                </span></a>
+                <a href="#" class="nav-items"><span>Item 4</span></a>
+            </div>
+        </nav>
+    </header>
+
+    <!-- main container -->
+
+    <div class="doc-container">
```

Comment:
For the main content, the `main` landmark element can be used. 

`aside` element can be nested inside the `main` element.